weather/models.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    ForeignKey,
+    DateTime,
+    Float,
+)
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "cities"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(63), nullable=False, unique=True)
+    additional_info = Column(String(255), )
```

Comment:
The `additional_info` column is defined without specifying `nullable=True`. By default, columns are nullable unless specified otherwise, but it's good practice to explicitly define nullability for clarity.